packages/components/test/index.js

```diff
@@ -241,15 +241,18 @@ describe('Box', () => {
 
     expect(json).toMatchSnapshot()
 
-    expect(json).toHaveStyleRule('color', '#fefefe')
-    expect(json).toHaveStyleRule('border', '2px solid #80e5ff')
-
-    expect(json).toHaveStyleRule('border-bottom-left-radius', '61px 8px')
-    expect(json).toHaveStyleRule('border-bottom-right-radius', '6px 68px')
+    expect(json).toHaveStyleRule(
```

Comment:
Oh that's much better.
Did I really write a test that randomly fails? :D

Actually, this goes into a snapshot, right? We should hardcode that "random" integer.